ebucore/amagi_ebucoreExtension.ttl

```diff
@@ -224,17 +226,33 @@
         rdfs:comment     "To express restriction on the number of times an asset can be published."@en ;
         rdfs:subClassOf  <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#Rights> .
 
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#hasPublicationCap>
+        rdf:type            rdf:Property ;
+        rdfs:range          <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#PublicationCap> ;
+        rdfs:domain         <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#PublicationRestrictions> ;
+        rdfs:comment        "Publication Caps."@en ;
+        rdfs:comment        "To express publication limits to airing"@en ;
+        rdfs:subPropertyOf  <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#isCoveredBy> ;
+        rdfs:label          "hasPublicationCap"@en .
+
+
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#PublicationCap>
+        rdf:type         rdfs:Class ;
+        rdfs:label       "PublicationCap"@en ;
+        rdfs:comment     "To provide information to restrict publications in a given interval of time"@en ;
+        rdfs:subClassOf  owl:Thing .
+
 <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#publicationCap>
     a               rdf:Property ;
     rdfs:range      <http://www.w3.org/2001/XMLSchema#string> ;
-    rdfs:domain     <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#PublicationRestrictions> ;
+    rdfs:domain     <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#PublicationCap> ;
     rdfs:label      "publicationCap"@en ;
     rdfs:comment    """To express restriction on total number of times the media can be published per interval. """@en .
 
 <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#publicationPeriod>
     a               rdf:Property ;
     rdfs:range      <http://www.w3.org/2001/XMLSchema#string> ;
-    rdfs:domain     <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#PublicationRestrictions> ;
+    rdfs:domain     <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#PublicationCap> ;
     rdfs:label      "publicationPeriod"@en ;
     rdfs:comment    """The publication period associated with the publicationCap for a given PublicationRestrictions instance. Period examples 6H, 1M, 2M, 3D, 1W, 10D, 1D6H, 1M15D where H->hours, M is month, D is day, W is week."""@en .
```

Comment:
@shashi-banger I hope publicationPeriod can be left empty if the publicationCap is for lifetime (irrespective of periods, overall lifetime cap)

Reply:
@ramamoorthy-amagi There is a separate property 

```
<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#maxPublicationEvents>
    a               rdf:Property ;
    rdfs:range      <http://www.w3.org/2001/XMLSchema#integer> ;
    rdfs:domain     <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#UsageRestrictions> ;
    rdfs:label      "maxPublicationEvents"@en ;
    rdfs:comment    "To express restriction on the maximum number of publication events for an asset in a given date range."@en .
```

The above can be used for lifetime cap